Tribes/src/app/building.ts

```diff
@@ -0,0 +1,9 @@
+export class Building {
+    id: number;
+    type: string;
+    level: number;
+    hp: number;
+    startedAt: number;
+    finishedAt: number;
+    imageURL?: string;
```

Comment:
if we give good names for our images in the assets, this is not needed because it will be
 some constant value: `src/assets/` + { building.type } + `.png` 
and it will work for all buildings and images

Reply:
done